model/build/build.go

```diff
@@ -327,14 +327,16 @@ func (b Builds) InsertMany(ctx context.Context, ordered bool) error {
 	return errors.Wrap(err, "bulk inserting builds")
 }
 
-// GetFinishedNotificationDescription returns a description of successful/failed tasks for the build,
-// to be used by jobs and notification processing.
-func (b *Build) GetFinishedNotificationDescription(tasks []task.Task) string {
+// GetPRNotificationDescription returns a GitHub PR status description based on
+// the statuses of tasks in the build, to be used by jobs and notification
+// processing.
+func (b *Build) GetPRNotificationDescription(tasks []task.Task) string {
 	success := 0
 	failed := 0
-	systemError := 0
```

Comment:
I removed this because `(task.Task).Status` only stores success/failed (the specific failure reasons in the UI are computed based on the task end details). Only tasks that are many years old store specific failure statuses like system-failed and whatnot.